src/styles/layout/_page.scss

```diff
@@ -0,0 +1,34 @@
+$tablet-size: 600px;
+$wide-screen-size: 1000px;
+
+.page {
+  margin: 0;
+  --main-white: #fff;
+  --main-dark: #161616;
+  --main-accent: #d12d35;
+  --main-text: #3e3e3e;
+  --secondary-accent: #ae1c23;
+  --form-background: #f8f8fa;
+  --form-label-grey: #7e7e83;
+
+  font-family: Cinzel, Georgia, Times, "Times New Roman", serif;
+  font-size: 16px;
+  color: var( --main-dark);
+  line-height: 27px;
+}
```

Comment:
add for your page scroll-behavior: smooth ; for smooth scroll to elements

Reply:
Did that as well